cmd/cli/cmd/alpha/manifest-gen/questions.go

```diff
@@ -0,0 +1,167 @@
+package manifestgen
+
+import (
+	"capact.io/capact/cmd/cli/cmd/alpha/manifest-gen/common"
+	"capact.io/capact/pkg/sdk/apis/0.0.1/types"
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/pkg/errors"
+)
+
+func askForManifestType() ([]string, error) {
+	var manifestTypes []string
+	availableManifestsType := []string{
+		string(types.AttributeManifestKind),
+		string(types.TypeManifestKind),
+		string(types.InterfaceGroupManifestKind),
+		string(types.InterfaceManifestKind),
+		string(types.ImplementationManifestKind),
+	}
+	prompt := []*survey.Question{
+		{
+			Prompt: &survey.MultiSelect{
+				Message: "Which manifests do you want to generate:",
```

Comment:
It would be great to have a minimal help to guide user how the Type toggle works:
```suggestion
				Message: "Which manifests do you want to generate:",
				Help:    "If the Interface is selected, Type toggles input and output Type generation for a given Interface.",
```